src/DataFixtures/SponsorshipLevelBenefitFixtures.php

```diff
@@ -25,25 +25,27 @@ public function __construct(SponsorshipLevelBenefitManagerInterface $sponsorship
 
     /**
      * @param ObjectManager $manager
+     *
      * @throws \Exception
      */
-    public function load(ObjectManager $manager)
+    public function load(ObjectManager $manager): void
     {
-        for ($sl = 0; $sl < SponsorshipLevelFixtures::NB_SPONSORSHIP_LEVEL; ++$sl) {
+        for ($sl = 0; $sl < SponsorshipLevelFixtures::NB_SPONSORSHIP_LEVEL; $sl++) {
```

Comment:
Pre to post = changes in index.
`$sl` is a bad name.

Reply:
Same as above about pre to post
I agree about $sb, i am going to fix it